.github/workflows/preview-release.yml

```diff
@@ -8,7 +8,8 @@ on:
 jobs:
     release:
         runs-on: ubuntu-latest
-
+        # Run the job if manually triggered or if the commit message includes a commit type of fix or feat or breaking change
+        if: contains(github.event.head_commit.message, 'fix:') || contains(github.event.head_commit.message, 'fix(') || contains(github.event.head_commit.message, 'feat:') || contains(github.event.head_commit.message, 'feat(') || contains(github.event.head_commit.message, '!:')
```

Comment:
using the `:` and `(` ensures it is a commit type and not just a random word in the message